### ISSUE TITLE
King Arthur: Knight's Tale (1157390)

### ISSUE THREAD

**Issue by WebsiteDeveloper** [ValveSoftware/Proton#5809]
# Compatibility Report
- Name of the game with compatibility issues: King Arthur: Knight's Tale
- Steam AppID of the game: 1157390

## System Information
- GPU: AMD Radeon RX 5700 XT
- Driver/LLVM version: 4.6 Mesa 22.1.0-rc3 navi10, LLVM 14.0.0, DRM 3.42
- Kernel version: 5.15.0-27-generic
- Link to full system information report as [Gist](https://gist.github.com/WebsiteDeveloper/d74ae6ef54f482ea43019a3ecb0c0d4b)
- Proton version: Experimental

[steam-1157390.log](https://github.com/ValveSoftware/Proton/files/9565163/steam-1157390.log)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Drop to Low Framereates usually after some Playtime

## Reproduction
Playing the game for about 30mins

**Comment by WebsiteDeveloper** [comment]
The Game runs fine otherwise. And there seems to be a correllation with a CPU Usage spike i cant 100% verify though

**Comment by WebsiteDeveloper** [comment]
According to another Report on protondb the performance drops alot when encountering grass or fog

**Comment by kisak-valve** [comment]
King Arthur: Knight's Tale

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5811.
@3DRaven posted on 2022-05-01T10:18:59:

# Compatibility Report
- Name of the game with compatibility issues: King Arthur: Knight's Tale
- Steam AppID of the game: 1157390

## System Information
- GPU: Nvidia 3080 RTX
- Driver/LLVM version: Nvidia 510.60.02
- Kernel version: 5.17.0-5.1-liquorix-amd64
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/3DRaven/9797bdce6442b535bc0099d943c70fdf
- Proton version: Experimental

## I confirm:
- [+] that I haven't found an existing compatibility report for this game.
- [+] that I have checked whether there are updates for my system available.


[steam-1157390.log](https://github.com/ValveSoftware/Proton/files/8598685/steam-1157390.log)


## Symptoms 
After start I hear sound and see cursor, but on black screen.

## Reproduction
Download and start the game.

**Comment by Blisto91** [comment]
@3DRaven it might be selecting your AMD apu. Try to see if launching with this helps.
```
__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only %command%
```

**Comment by 3DRaven** [comment]
No, It not works. New proton log in [steam-1157390.log](https://github.com/ValveSoftware/Proton/files/8625312/steam-1157390.log)

**Comment by 3DRaven** [comment]
I used DXVK_HUD=full and took screenshot in


![Снимок экрана от 2022-05-04 20-18-19](https://user-images.githubusercontent.com/3204467/166800307-6943fb6a-5372-47ba-a3e1-dd48e3edd852.png)

**Comment by lemba23** [comment]
# Compatibility Report
- Name of the game with compatibility issues: King Arthur: Knight's Tale
- Steam AppID of the game: 1157390

## System Information
- GPU: RTX 2060 Super
- Driver/LLVM version: Nvidia Driver Version: 510.60.02
- Kernel version: 5.17.0-5.1-liquorix-amd64
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/lemba23/cc3b7a47ab0d60589327a6b9a5b5b581
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1157390.log](https://github.com/ValveSoftware/Proton/files/8644944/steam-1157390.log)


## Symptoms 
Grass and bushes "smear" all over the screen.
![20220507122405_1](https://user-images.githubusercontent.com/5611211/167250944-8139bcec-1606-4092-883c-d8f733143404.jpg)


## Reproduction
Occurs every time, when grass or bushes are rendered.

**Comment by WebsiteDeveloper** [comment]
Currently works with Proton 6.3.8
Frequent stutters with Proton 7 and up

**Comment by Blisto91** [comment]
Is the weird bush and grass thing above still an issue or is it just performance or stutters?

**Comment by WebsiteDeveloper** [comment]
I haven't had any artifacts. Only slowly degrading performance to near unplayable input lag and 5-10 fps with proton 7 and experimental

**Comment by kisak-valve** [comment]
Hello @WebsiteDeveloper, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by WebsiteDeveloper** [comment]
@kisak-valve Added both to the original Report
